tests/backup/backup_helper.go

```diff
@@ -76,6 +79,11 @@ type userAccessContext struct {
 	context  context.Context
 }
 
+type BackupRestoreContext struct {
+	// schedCtxs is an array of `*scheduler.Context`s (which correspond to namespaces), each of which contain various K8s objects (speclist), that together make up the backup.
+	schedCtxs []*scheduler.Context
+}
+
```

Comment:
Not sure why this struct was required. You could have used it directly as a slice right?

Reply:
As part of deeper validation in a future PR, I'll be adding more fields to this struct. Otherwise I would've used a "typedef" as having something like `[]*[]*schedu...` looks confusing as to what it actually means (which is what it would look like, as currently, I maintain an array of `backupRestoreContext`).